services/core-api/app/api/services/nros_now_status_service.py

```diff
@@ -41,17 +43,28 @@ def nros_now_status_update(cls, now_number, status, status_updated_date):
                 'lobName': 'MMS'
             },
             data=json.dumps(data))
-        current_app.logger.info(
-            f'Status for NoW Number{now_number} pushed to NROS with response: {resp.__dict__}')
+        try:
+            resp.raise_for_status()
+            current_app.logger.info(
+                f'Status for NoW Number{now_number} pushed to NROS with response: {resp.__dict__}')
+        except:
+            current_app.logger.info(f'Failed to push status to NROS. Response: {resp.__dict__}')
+            pass
+
         return
 
     @classmethod
     def get_nros_now_token(cls):
         auth = HTTPBasicAuth(cls.nros_now_token_user, cls.nros_now_token_pass)
         resp = requests.get(cls.nros_now_token_url, auth=auth)
+        token = None
+        try:
+            resp.raise_for_status()
+            resp_body = json.loads(resp.text)
 
-        resp_body = json.loads(resp.text)
-
-        token = resp_body["access_token"]
+            token = resp_body["access_token"]
+        except:
+            current_app.logger.info(f'Failed to get a token from NROS. Response: {resp.__dict__}')
+            pass
```

Comment:
I think the `pass` can be removed.

Reply:
I dont think its needed but I put it in just to show it was a deliberate choice to ignore the error.